test/utils/deployment.ts

```diff
@@ -168,14 +228,38 @@ async function waitForApiStartupAsync(logStream: ChildProcessWithoutNullStreams)
         logStream.stdout.on('data', (chunk: Buffer) => {
             const data = chunk.toString().split('\n');
             for (const datum of data) {
-                if (/API (HTTP) listening on port 3000!/.test(datum)) {
+                if (/API \(HTTP\) listening on port 3000!/.test(datum)) {
```

Comment:
suggestion (non-blocking): I'd rather sleep then rely on  log output 😆. Or check for port recv